From 31f84439fa7803bdec22da432dda2c2e2c358912 Mon Sep 17 00:00:00 2001
From: Rishi Gupta <rishgupt@codeaurora.org>
Date: Sun, 23 Aug 2020 20:12:29 +0530
Subject: [PATCH 2/3] ima: disable creating sysfs entries for ima policy

The rules to measure and appraise files are now embedded
in driver file ima-policy.c itself. Therefore there is no
need for sysfs files that are required to send new rules
to kernel. This commit removes call to the logic to create
those files. Additionally, it increases security a bit also.

Change-Id: I0cea009347073dbb9b377c742325ed0f12d25f25
Signed-off-by: Rishi Gupta <rishgupt@codeaurora.org>
---
 security/integrity/ima/ima_init.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/security/integrity/ima/ima_init.c b/security/integrity/ima/ima_init.c
index 2967d49..ac507cf 100644
--- a/security/integrity/ima/ima_init.c
+++ b/security/integrity/ima/ima_init.c
@@ -137,5 +137,5 @@ int __init ima_init(void)
 
 	ima_init_policy();
 
-	return ima_fs_init();
+	return 0;
 }
-- 
2.7.4

